tool/RELEASE_INSTRUCTIONS.md

```diff
@@ -352,20 +352,20 @@ onto the `flutter/devtools` protected branch (`master`).
    in **zero** lines of code.
    - Ask a member of the DevTools team for review, but **DO NOT** squash and merge yet.
 
-2. Contact a member of the Dash team who has Admin access to the `flutter/devtools`
-repository settings (@godofredoc or @devoncarew). Ask them to:
-    - temporarily modify the `flutter/devtools` repository settings to "allow merge
-      commits at the repo level and remove `require linear history`". 
-    
-   Provide them with a link to your PR for context.
+2. Create internal bug using http://go/flutter-security-bug, with priority P2 and type `Process`:
```

Comment:
bonus, pre-fill this issues priority, type, content, title, etc. using query params in the URL